Libraries/src/Amazon.Lambda.Serialization.Json/AwsResolver.cs

```diff
@@ -96,7 +96,7 @@ protected override IList<JsonProperty> CreateProperties(Type type, MemberSeriali
                     }
                 }
             }
-            else if (type.FullName.Equals("Amazon.DynamoDBv2.Model.AttributeValue", StringComparison.Ordinal))
+            else if (type.FullName.Equals("Amazon.Lambda.DynamoDBEvents.DynamoDBEvent+AttributeValue", StringComparison.Ordinal))

```

Comment:
Is there a reason we can't leave the old `else if`? That way if somebody updates `Amazon.Lambda.Serialization.Json` but not the event package they wouldn't get a break in behavior.